### PR TITLE
adding get call

### PR DESCRIPTION
**Pull request by chelpu**
one issue is that I'm parsing duration into a string because json.Unmarshal has issues marshalling "" into an int. if I add a string annotation I get this error: "json: invalid use of ,string struct tag". if I don't add the annotation I get a twilio error. we can talk more in person.

### CODE REVIEW

**Comment by natebrennand** [comment]
try making it a pointer to an int?

File: common/response.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func (j *JsonTime) UnmarshalJSON(b []byte) error {
 type ResponseCore struct {
 	AccountSid   string   `json:"account_sid"`
 	ApiVersion   string   `json:"api_version"`
-	Body         string   `json:"body"`
+	Body         string   `json:"body"` //why is this here
```
Comment (natebrennand) [inline common/response.go]: because i made a mistake, feel free to move it to Message in sms

File: examples/get_call/main.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import (
+	"bufio"
+	"fmt"
+	"github.com/natebrennand/twiliogo"
+	"os"
+)
+
+func main() {
+	fmt.Println("Waiting for sid to send request")
+	sid, _ := bufio.NewReader(os.Stdin).ReadString('\n') // wait to let server catching callback start
```
Comment (natebrennand) [inline examples/get_call/main.go]: you're not using a callback, you don't need this stuff

Comment (natebrennand) [inline examples/get_call/main.go]: oops, only read the comment, not the code. code is right

File: voice/calls.go [hunk 3]
```diff
@@ -116,3 +116,19 @@ func (act VoiceAccount) Call(p Post) (Response, error) {
...
+	// send get request to twilio
+	return common.SendGetRequest(destUrl, act, resp, 200)
+}
+
+func (act VoiceAccount) Get(sid string) (Response, error) {
+	if true != validateCallSid(sid) {
+		return Response{}, errors.New("Invalid sid")
+	}
+
+	var m Response
```
Comment (natebrennand) [inline voice/calls.go]: define this at the start of the method then return it if error? It'll still be empty but looks nice

**Comment by natebrennand** [comment]
lgtm besides comments. Can you move the bad attributes outside of common or open an issue and I'll add it to other sms stuff?

File: voice/calls.go (hunk 3 above)
Comment (natebrennand) [inline voice/calls.go]: like this:

``` go
var m Response
if true != validateCallSid(sid) {
    return m, errors.New("Invalid sid")
}
err := act.getCall(fmt.Sprintf(getUrl, act.AccountSid, string(sid)), &m)
return m, err
```

**Comment by chelpu** [comment]
Done, except I still can't figure out the duration thing... I tried a *int and that wasn't working.

**Comment by natebrennand** [comment]
we can try messing with it tomorrow I guess. I don't think go's json lib handles `null` very well. We might need to do something like the `JsonPrice` class

**Pull request merged**